takwimu/takwimu_ui/src/components/AnalysisContent/Actions/DownloadPDF.js

```diff
@@ -235,8 +235,10 @@ function DownloadPDF({ classes, title, topic, data, takwimu }) {
 
   return (
     <ButtonBase
-      disableRipple
-      disableTouchRipple
+      ga-on="click"
+      ga-event-category="Analysis"
+      ga-event-action={`Download ${title}`}
+      ga-event-label={title}
```

Comment:
Same as above, does `title` included country name or code?

Reply:
Is it possible to include a number in `ga-event-value`, say 0 and 1 to indicate whether the button clicked was the one at the top of analysis page or bottom? Since we have that `boolean` to either show date, we can use it to know which number to set.

Reply:
I think its possible, we also have max scroll so we know if users scrolled to bottom of page.